app/classifier/drawing-tools/index.coffee

```diff
@@ -14,4 +14,5 @@ module.exports =
   point: require './point'
   polygon: require './polygon'
   rectangle: require './rectangle'
+  rotateRectangle: require('./rotate-rectangle.jsx').default
```

Comment:
Doesn't this need the `.jsx` extension? We don't use it anywhere else that we require JSX components into CJSX files.

Reply:
Good catch, this is a hold over from when I was converting out of `cjsx`, I will remove it.

Reply:
Cool. I'll hold off on merging until you've updated the code.